angularjs-portal-home/src/main/webapp/my-app/search/partials/search-results.html

```diff
@@ -5,23 +5,28 @@
         <a>All ({{ totalCount }})</a>
       </li>
       <li ng-click="filterTo('myuw')" id="myuw-selector">
-        <a>MyUW ({{myuwFilteredResults.length}})</a>
+        <a>MyUW <span ng-hide="portletListLoading">({{myuwFilteredResults.length}})</span>
+        <i ng-show="portletListLoading" class="fa fa-spinner fa-pulse"></i></a>
```

Comment:
Interesting design decision. Do we think 2 loading gifs for the same section may be too much (one in the body, one in the tab)? @keirserrie ? I personally could go either way, maybe i'd have to see it in slow mo.